src/_layouts/default.html

```diff
@@ -25,6 +25,13 @@
     <link rel="icon" type="image/png" sizes="16x16" href="/favicon-16x16.png" />
 
     <link rel="preconnect" href="https://fonts.gstatic.com" />
+    <link
+      href="https://cdn.jsdelivr.net/npm/bootstrap@5.2.3/dist/css/bootstrap.min.css"
+      rel="stylesheet"
+      integrity="sha384-rbsA2VBKQhggwzxH7pPCaAqO46MgnOM80zW1RWuH61DGLwZJEdK2Kadq2F9CUG65"
+      crossorigin="anonymous"
+    />
+
     <link href="https://fonts.googleapis.com/css2?{{site.fonts}}&display=swap" rel="stylesheet" />
     <link
```

Comment:
Giving this a closer look: I _think_ the animate.css file can also be removed. https://animate.style/ I don't see any CSS classes that start with`animate` in the codebase.

Reply:
CaGov.ttf file can also be removed.